### PR TITLE
build: release v6.45.0

### PR DESCRIPTION
**Pull request by LinHuiqing**
## New

- chore: update payment guide in payment unsupported msg [`#6199`](https://github.com/opengovsg/FormSG/pull/6199)
- fix(deps): bump type-fest from 3.8.0 to 3.9.0 in /shared [`#6195`](https://github.com/opengovsg/FormSG/pull/6195)
- chore: update previous payment typing and hooks [`#6191`](https://github.com/opengovsg/FormSG/pull/6191)
- chore: add missed out changelog for 6.44.1 [`#6197`](https://github.com/opengovsg/FormSG/pull/6197)
- build: merge release v6.44.1 to develop [`#6196`](https://github.com/opengovsg/FormSG/pull/6196)
- build: release v6.44.1 [`#6193`](https://github.com/opengovsg/FormSG/pull/6193)
- perf: memoize field row containers [`#6189`](https://github.com/opengovsg/FormSG/pull/6189)
- build: merge release v6.44.0 to develop [`#6187`](https://github.com/opengovsg/FormSG/pull/6187)

### chore: update payment guide in payment unsupported msg [`#6199`](https://github.com/opengovsg/FormSG/pull/6199)
@KenLSM 

- [x] go to an email mode form. Go to settings, make sure that the settings page is as seen in the screenshot
- [x] go to the payment guide link, make sure that it can open 

### chore: update previous payment typing and hooks [`#6191`](https://github.com/opengovsg/FormSG/pull/6191)
@KenLSM 

Previous payment check should still work correctly, duplicating test cases from #6170 
**To check basic functionality**
Go to a payment form
- [x] make a payment successfully
- [x] attempt to make a second payment with the same email
- [x] you should see the modal directing you to your previous payment page
- [x] click the link and ensure you are the same payment page from when you made a successful payment

**To check that we get the latest successful payment**
all of these are done with the same email as above
- [x] go back and make a second payment with the same email again (take note of your payment id/payment page link)
- [x] make a payment successfully
- [x] go back to the form
- [x] make a payment intent (but do not pay)
- [x] go back to the form
- [x] fill in the form and click proceed to pay, the duplicate payment modal should show, directing you to the **second** payment that you made. Check that the payment id/payment page link is equivalent to your second payment

**To check that we do not erroneously show the duplicate payment modal**
On the same form
- [x] make a payment with a different email
- [x] you should not see the duplicate payment modal at proceed to pay

On a different form
- [x] make a payment with the original email form above
- [x] you should not see the duplicate payment modal at proceed to pay

**To ensure no regression issues on non-payment forms**
(As changes was made to PublicFormSubmitButton)
- [x] go to a non-payment storage mode form
- [x] make a submission
- [x] go to an email mode form
- [x] make a submission

### perf: memoize field row containers [`#6189`](https://github.com/opengovsg/FormSG/pull/6189)
@LinHuiqing 

**To ensure the feature works**

(skipped: react profiler only works in development)
On development environment: Use react dev tool -> profiler
- [ ] go to a form in your admin view
- [ ] start profiler recording
- [ ] shift some of the fields around
- [ ] stop recording, you should see that most fields are not re-rendered at each step, and they are memoized at the FieldRow level

On non-development environment: Use react dev tool -> components
- [x] go to a form in your admin view
- [x] each question should be denoted by a `Hr` and have the `Memo` tag on the right of it

Go to a large form
- [x] try to drag a field. The dragging of the field (before dropping) should feel smooth, and there should be minimal jitters.

**To ensure core features are still intact**
- [x] In any storage form, ensure the following functionality is still working
    - [x] Create a new field
    - [x] Edit the field properties of any field
    - [x] Edit the field properties, do not save, click away (close the drawer or select another field) the discard editing modal should appear
    - [x] Drag a field to change its position, question number should be updated too
    - [x] Duplicate a field
    - [x] Attempt to delete a field, the modal should open, and can be cancelled
    - [x] Delete the field, modal should open, and field can be deleted
    - [x] Preview the form. Make a submission
    - [x] Open the form to respondents. Make a submission
- [x] Do the same steps above for an email form
- [x] Do the same steps on mobile 

#### Notes
There are still a few unnecessary re-renders upon drag and dropping of the fields.

Ideally, only the fields affected (field id changed) should re-render and all other fields should be unaffected. However, currently there is still some existing hooks that causes a re-render for all the fields even if they are memoized

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,35 @@ All notable changes to this project will be documented in this file. Dates are d
...
 
+#### [v6.45.0](https://github.com/opengovsg/FormSG/compare/v6.45.0...v6.45.0)
+
+- chore: update payment guide in payment unsupported msg [`#6199`](https://github.com/opengovsg/FormSG/pull/6199)
+- fix(deps): bump type-fest from 3.8.0 to 3.9.0 in /shared [`#6195`](https://github.com/opengovsg/FormSG/pull/6195)
+
+#### [v6.45.0](https://github.com/opengovsg/FormSG/compare/v6.44.1...v6.45.0)
+
+> 26 April 2023
+
```
Comment (LinHuiqing) [inline CHANGELOG.md]:
```suggestion
#### [v6.45.0](https://github.com/opengovsg/FormSG/compare/v6.44.1...v6.45.0)

> 26 April 2023

- chore: update payment guide in payment unsupported msg [`#6199`](https://github.com/opengovsg/FormSG/pull/6199)
- fix(deps): bump type-fest from 3.8.0 to 3.9.0 in /shared [`#6195`](https://github.com/opengovsg/FormSG/pull/6195)
```

**Comment by foochifa** [comment]
@LinHuiqing I forgot profiler doesn't work for non dev env HAHAHA instead, you could go to react dev tools -> components. And make sure the fieldrowcomponent (as a Hr I think) is memo-ed (so there should be a memo tag to it)

**Comment by LinHuiqing** [comment]
> ...field id should be updated too.

do you mean question number? @foochifa

**Comment by foochifa** [comment]
Ah yes field id is question number hahah sorry for the confusion

**Comment by LinHuiqing** [comment]
just to note for https://github.com/opengovsg/FormSG/pull/6189: the form and drag + drop is noticeably laggier on mobile

**Review by KenLSM — APPROVED** [review]
lgtm!

**Pull request merged**